src/frontend/src/optimizer/rule/table_function_to_internal_backfill_progress.rs

```diff
@@ -58,7 +58,7 @@ impl FallibleRule<Logical> for TableFunctionToInternalBackfillProgressRule {
 
 impl TableFunctionToInternalBackfillProgressRule {
     fn build_plan(
-        ctx: Rc<OptimizerContext>,
+        ctx: Arc<OptimizerContext>,
```

Comment:
Can use `OptimizerContextRef` in case we want to change the type again in the future. Can simply replace keyword `Arc<OptimizerContext>` with `OptimizerContextRef.`